fence/job/visa_update_cronjob.py

```diff
@@ -0,0 +1,177 @@
+import asyncio
+import datetime
+import time
+
+from cdislogging import get_logger
+from userdatamodel.driver import SQLAlchemyDriver
+
+from fence.config import config
+from fence.models import (
+    GA4GHVisaV1,
+    User,
+    UpstreamRefreshToken,
+    query_for_user,
+)
+from fence.resources.openid.ras_oauth2 import RASOauth2Client as RASClient
+
+
+logger = get_logger(__name__, log_level="debug")
+
+
+class Visa_Token_Update(object):
+    def __init__(
+        self,
+        chunk_size=None,
+        concurrency=None,
+        thread_pool_size=None,
+        buffer_size=None,
+        logger=logger,
+    ):
+        """
+        args:
+            chunk_size: size of chunk of users we want to take from each iteration
+            concurrency: number of concurrent users going through the visa update flow
+            thread_pool_size: number of Docker container CPU used for jwt verifcation
+            buffer_size: max size of queue
+        """
+        self.chunk_size = chunk_size or 10
+        self.concurrency = concurrency or 5
+        self.thread_pool_size = thread_pool_size or 3
+        self.buffer_size = buffer_size or 10
+        self.n_workers = self.thread_pool_size + self.concurrency
+        self.logger = logger
+
+        self.visa_types = config.get("USERSYNC").get("visa_types")
```

Comment:
```suggestion
        self.visa_types = config.get("USERSYNC", {}).get("visa_types", [])
```

Reply:
```suggestion
        self.visa_types = config["USERSYNC"].get("visa_types", {})
```

or this, because the values should never be empty as long as they are in the default config (see https://github.com/uc-cdis/gen3config if you're interested)